README.md

```diff
@@ -43,6 +43,7 @@ or Yarn
     -d, --date             CHANGELOG.md release date in the form of a valid date
                            string. Uses system new Date([your date])
                                     [string] [default: "2022-12-17T02:28:23.456Z"]
+    -f, --file             changelog output file[string] [default: "CHANGELOG.md"]
```

Comment:
required:
update the whole README.md help output, we made updates to everything